src/Sylius/Bundle/CoreBundle/Resources/views/Email/accountVerification.html.twig

```diff
@@ -1,3 +1,5 @@
+{% deprecated 'The "@SyliusCore/Email/accountVerification.html.twig" template is deprecated since Sylius 1.8 and will be removed in Sylius 2.0, use "@SyliusCore/Email/verification.html.twig" instead.' %}
```

Comment:
You can remove all the other lines from here and just extend the new template

Reply:
I'm uncertain as the new template lacks the `sylius_bundle_loaded_checker` logic and the translation keys have also changed